### ISSUE TITLE
how is train set constructed? Is the data open sourced?

### ISSUE THREAD

**Issue by DryPilgrim** [Yifan-Song793/RestGPT#7]
//cn
1.在data collection中没有提到train set，它是怎么构造的？数据有开源吗？
2.data collection中提到的development set和test set都是用来评估模型的吗？
3.test set总共只有157条数据，如何确保测试结果的可靠性？
//en
1. there is no mention of the TRAIN SET in the DATA COLLECTION, how is it constructed? Is the data open source?
2. Are both the development set and test set mentioned in the data collection used to evaluate the model?
3. how to ensure the reliability of the test results when the test set has only 157 data in total?

**Comment by Davo00** [comment]
@DryPilgrim From my understanding, this is strictly prompt based approach, there is no need of a fine-tuning, therefore a train set does not exist
